FlowCrypt/src/main/java/com/flowcrypt/email/database/entity/MessageEntity.kt

```diff
@@ -176,16 +184,13 @@ data class MessageEntity(
     parcel.writeString(threadId)
     parcel.writeString(historyId)
```

Comment:
Do you use message `historyId` property anywhere in the app? As on iOS it's currently not used

Reply:
Yup. We use it to sync the last changes. Details https://developers.google.com/gmail/api/reference/rest/v1/users.history/list